internal/pkg/miniprocfs/processes.go

```diff
@@ -164,7 +164,7 @@ func (procs *Processes) readProc(pidString string) (*machine.ProcessInfo, error)
 	var label string
 
 	if err = procs.readFileIntoBuf(path + "attr/current"); err == nil {
-		label = string(bytes.Trim(procs.buf, "\x000\n"))
+		label = string(bytes.Trim(procs.buf, "\x00\n"))
```

Comment:
we need to strip it in the CLI then, as `\0` is not a valid output

Reply:
see #9564

Reply:
I strip `\x00` == `\0` in C convention. `\x000` is like `\x00` and `0`

Reply:
yeh, my mistake should have been just `\x00`